### DISCUSSION TITLE
Usage of Kalman filter

### DISCUSSION THREAD

**Discussion by krober10nd** [anitagraser/movingpandas#285] [Q&A] [answered]
On the readme it's stated that Kalman filters are used to "clean" the trajectories. Can you show me where in the source code this is done? Thanks

**Comment by anitagraser** [comment, marked as answer]
The Kalman filter is implemented in the  KalmanSmootherCV class

https://github.com/anitagraser/movingpandas/blob/1c93777b0e6cf2f44f063b6bc788964bee6df163/movingpandas/trajectory_smoother.py#L88-L95
